### PR TITLE
Add events across teams with page support

### PR DESCRIPTION
**Pull request by marquezzy**

### CODE REVIEW

File: TeamSnapSDK/SDK/DataLayer/TSDKObjectsRequest.h [hunk 1]
```diff
@@ -28,4 +28,6 @@
 + (NSArray *_Nonnull)knownCompletionTypes;
 + (void)dumpCompletionTypes;
 
++ (void)listEventsForTeams:(NSArray * _Nonnull)teams pageNumber:(NSNumber *_Nullable)pageNumber pageSize:(NSNumber *_Nullable)pageSize startDate:(NSDate*_Nullable)startDate endDate:(NSDate*_Nullable)endDate completion:(TSDKArrayCompletionBlock _Nullable)completion;
```
Comment (jrahaim) [inline TeamSnapSDK/SDK/DataLayer/TSDKObjectsRequest.h]: - [ ] This should have an object type for the items in the teams Array.  (is it looking for IDs? TSDKTeams?)
(if it's looking for TeamIds I might even rename the method to listEventsForTeamIds
- [ ] The return type should probably be TSDKEventArrayCompletionBlock

Comment (marquezzy) [inline TeamSnapSDK/SDK/DataLayer/TSDKObjectsRequest.h]: Fixed these two issues.

File: TeamSnapSDK/SDK/DataLayer/TSDKObjectsRequest.m [hunk 2]
```diff
@@ -209,16 +209,16 @@ + (void)listEventsForTeam:(TSDKTeam *)team startDate:(NSDate *)startDate endDate
     }
 }
 
-+ (NSDictionary *)eventsParametersWithTeams:(NSArray *)teams pageNumber:(NSNumber *)pageNumber pageSize:(NSNumber *)pageSize startDate:(NSDate*)startDate endDate:(NSDate*)endDate {
++ (NSDictionary *)eventsParametersWithTeams:(NSArray *)teams pageNumber:(NSInteger)pageNumber pageSize:(NSInteger)pageSize startDate:(NSDate*)startDate endDate:(NSDate*)endDate {
     
     NSString *teamIds = [self teamIdsParameterForTeams:teams];
     NSMutableDictionary *paramaters = [[NSMutableDictionary alloc] init];
     [paramaters setValue:teamIds forKey:@"team_id"];
     if (pageNumber) {
```
Comment (jrahaim) [inline TeamSnapSDK/SDK/DataLayer/TSDKObjectsRequest.m]: I am surprised this compiled, but this check can be removed.

File: TeamSnapSDK/SDK/DataLayer/TSDKObjectsRequest.m [hunk 3]
```diff
@@ -209,16 +209,16 @@ + (void)listEventsForTeam:(TSDKTeam *)team startDate:(NSDate *)startDate endDate
...
-        [paramaters setValue:[pageNumber stringValue] forKey:@"page_number"];
+        [paramaters setValue:[@(pageNumber) stringValue] forKey:@"page_number"];
     }
     if (pageSize) {
```
Comment (jrahaim) [inline TeamSnapSDK/SDK/DataLayer/TSDKObjectsRequest.m]: Can also be removed.

**Pull request merged**